package/yast2-packager.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Sep 14 14:30:50 UTC 2017 - jreidinger@suse.com
+
+- show product path in url for multi product medias
```

Comment:
Bad English: singular is `medium`, plural is `media`
(Fix the line below as well.)